### DISCUSSION TITLE
Is there no way to change order of  the organizations displayed above Activity overview in my profile?

### DISCUSSION THREAD

**Discussion by donkeeman** [community/community#19038] [Other Feature Feedback, Questions, & Ideas]
In activity overview, it shows only 3 organizations and the others are hidden in 'more'.
I want to show the organization that contributed more and did more activities but I can't find how to customize or change order.
Is it impossible?

**Comment by melvincwng** [comment]
Hi @donkeeman, unfortunately I don't think there's an option to change the order of organizations in your Activity overview.

Hope this clarifies.
